blog/2024-01-08/index.md

```diff
@@ -5,13 +5,18 @@ authors: [razze, bertob]
 tags: [flathub, quality]
 ---
 
-Flathub has grown massively over the past few years, with over 2400 apps at present. In the early days our goal was mostly to make it easy to get the latest version of popular apps via Flatpak. Today we can safely say that this goal has been achieved!
+Flathub has grown massively over the past few years, making it easier than ever to get a ton of apps all from one place—but with that growth comes new problems. We're here with a solution to ensure developers' apps, Flathub, and the entire Linux desktop ecosystem is presented in what we believe is the best light possible.
 
-Not only can you get pretty much any app on Flathub, they are often directly maintained by the developer, and it's the primary channel many new app developers choose for shipping their apps.
+<!-- truncate -->
+
+## Growth
```

Comment:
Feels a bit boring?

Reply:
Yeah honestly it could just be omitted; the following paragraph works fine with the preceding one.

```suggestion
```